pennylane/transforms/dynamic_one_shot.py

```diff
@@ -19,7 +19,7 @@
 
 # pylint: disable=import-outside-toplevel
 from collections import Counter
-from typing import Callable, Sequence
+from typing import Callable, Sequence, Tuple
```

Comment:
To follow up on the above comment.

```suggestion
from typing import Callable, Sequence
```